### PR TITLE
feat(@angular-ru/common): add CoerceBoolean pipe

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [ ] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: packages/common/docs/pipes.md [hunk 1]
```diff
@@ -749,3 +749,25 @@ export class AppComponent {
...
+@NgModule({
+    // ..
+    imports: [ToBooleanPipeModule]
+})
+export class AppModule {}
+
+@Component({
+    //...
+    template: `
+        <mat-select [multiple]="multiple | toBoolean">first</p>
```
Comment (Schirbak) [inline packages/common/docs/pipes.md:767]: maybe call it `coerceBoolean`, as internal function?

Comment (splincode) [reply packages/common/docs/pipes.md:767]: image

Maybe, but I think we should align naming with another converters

Comment (Schirbak) [reply packages/common/docs/pipes.md:767]: i mean, coerceBoolean and toBoolean is different things. 
`toBoolean = val => Boolean(val)`
`coerceBoolean = val => val !== '' ...`

but if you want to forgive the dumb logic of falsy values, i will support it.

Comment (splincode) [reply packages/common/docs/pipes.md:767]: renamed

**Review by nikitasadchikov — APPROVED** [review]

**Comment by splincode** [comment]
@Schirbak friendly ping

**Review by Schirbak — APPROVED** [review]

**Pull request merged**